Parent:     0b777b00 (Kits: Always use settings found in the SDK)
Author:     Ray Donnelly <mingw.android@gmail.com>
AuthorDate: 2012-08-06 17:58:19 +0000
Commit:     Ray Donnelly <mingw.android@gmail.com>
CommitDate: 2012-12-13 12:38:17 +0000

Fix caplitalization for MinGW-w64 cross compilation.

MinGW-w64 cross compilation packages always use lower-case filenames.
library name "Ws2_32" changed to "ws2_32"
include name "Windows.h" changed to "windows.h"

Change-Id: Id5e683d0a943a32d20476f4b710bf2e5ee0bbb75
diff --git a/src/tools/win64interrupt/win64interrupt.c b/src/tools/win64interrupt/win64interrupt.c
index acda592..f8f1840 100644
--- a/src/tools/win64interrupt/win64interrupt.c
+++ b/src/tools/win64interrupt/win64interrupt.c
@@ -35,7 +35,7 @@
 #error Must target Windows NT 5.0.1 or later for DebugBreakProcess
 #endif
 
-#include <Windows.h>
+#include <windows.h>
 #include <stdio.h>
 
 /* To debug break a 64bit application under Windows, you must call
diff --git a/tests/manual/utils/tcpportsgatherer/tcpportsgatherer.pro b/tests/manual/utils/tcpportsgatherer/tcpportsgatherer.pro
index 1c0a7f4..0e50083 100644
--- a/tests/manual/utils/tcpportsgatherer/tcpportsgatherer.pro
+++ b/tests/manual/utils/tcpportsgatherer/tcpportsgatherer.pro
@@ -24,5 +24,5 @@
     $${UTILSDIR}/portlist.cpp \
     $${UTILSDIR}/tcpportsgatherer.cpp
 
-win32:LIBS += -liphlpapi -lWs2_32
+win32:LIBS += -liphlpapi -lws2_32
 SOURCES += main.cpp
diff --git a/tests/manual/utils/tcpportsgatherer/tcpportsgatherer.qbp b/tests/manual/utils/tcpportsgatherer/tcpportsgatherer.qbp
index 4a491ff..c84e9fd 100644
--- a/tests/manual/utils/tcpportsgatherer/tcpportsgatherer.qbp
+++ b/tests/manual/utils/tcpportsgatherer/tcpportsgatherer.qbp
@@ -16,7 +16,7 @@
 
     Properties {
         condition: qbs.targetOS == "windows"
-        cpp.dynamicLibraries: [ "iphlpapi.lib", "Ws2_32.lib" ]
+        cpp.dynamicLibraries: [ "iphlpapi.lib", "ws2_32.lib" ]
     }
 
     Depends { name: "cpp" }
